src/index.js

```diff
@@ -1,4 +1,9 @@
 #! /usr/bin/env node
-console.log('Hello, Hammer!')
 
-// main logic of cli should go here
+import run from './cli';
+
+function main() {
+  run();
+}
+
+main();
```

Comment:
This can be re-written as-
`(function main() {
run();
})();`

Reply:
Doesn't pass linter. Not implementing.